bin/cross-compile

```diff
@@ -15,3 +15,5 @@ timestamp=$(TZ=UTC date -u '+%Y-%m-%dT%H:%M:%SZ')
 sha=$(git rev-parse HEAD)
 gox -ldflags "-X github.com/Originate/git-town/cmd.Version=$TRAVIS_TAG -X github.com/Originate/git-town/cmd.BuildTime=$timestamp) -X github.com/Originate/git-town/cmd.GitHash=$sha" \
     -output "dist/{{.Dir}}-{{.OS}}-{{.Arch}}"
+
+package/debian/make_deb.sh
```

Comment:
Thoughts on creating a new script: `bin/build-release` which has runs `bin/cross-compile` and `package/debian/make_deb.sh`. I don't like building a debian package being in the `cross-compile` script

Reply:
I like it. This will also help out if we end up building for other distros in the future